include/openssl/tls1.h

```diff
@@ -323,6 +323,12 @@ __owur int SSL_check_chain(SSL *s, X509 *x, EVP_PKEY *pk, STACK_OF(X509) *chain)
 # define SSL_set_tlsext_status_ocsp_resp(ssl, arg, arglen) \
         SSL_ctrl(ssl,SSL_CTRL_SET_TLSEXT_STATUS_REQ_OCSP_RESP,arglen,arg)
 
+# define SSL_get_tlsext_status_ocsp_resp_ex(ssl, arg) \
```

Comment:
This function returns a reference to volatile internal data that is still owned by the library. The name should reflect this (...get0...) and its documentation should make this clear (both the stack and its content belong the SSL connection object and MUST NOT be freed by the caller).
```suggestion
# define SSL_get0_tlsext_status_ocsp_resp_ex(ssl, arg) \
```

Reply:
I will change it. That will also mean to rename `SSL_set_tlsext_status_ocsp_resp` to ` SSL_set0_tlsext_status_ocsp_resp`

Correct?